src/client/reducers/transactionsReducer.js

```diff
@@ -1,20 +1,15 @@
-
-import { sampleData } from '../../../sampleData';
 import * as types from '../constants/actionTypes';
 
 let nextTransactionId = 0;
-
 const addIndex = object => ({
   ...object, 
   id: nextTransactionId++
 });
-
-
 export const transactions = (state = [], action) => {
   switch  (action.type) {
     case types.ADD_TRANSACTION: 
       const incoming = action.payload.map(trans => addIndex(trans))
-      return [...state, ...incoming]
+      return [...incoming, ...state]
```

Comment:
if we push new transactions to the front without modifying the selection index, won't we change the displayed transaction every time one comes in?